recipes/pangu/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "565bd4c799be35d77af3622c9e3264739040644bd27783814462b2b4378b5c24" %}
+
+package:
+  name: pangu
+  version: {{ version }}
+
+source:
+  url: https://github.com/PacificBiosciences/pangu/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pangu = pangu.main
+  script: python -m pip install --no-deps -vv .
+
+requirements:
+  host:
+    - python >=3.9
```

Comment:
The `host:` section should only list dependencies that are needed for running the `pip install` step, so in this case, probably just `python` and `pip` (since your `setup.py` doesn’t import any external packages).